docs/releases/saros-e_16.0.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Saros for Eclipse 16.0.0 Release Notes
+---
+
+:tada: We are happy to announce **Saros for Eclipse 16.0.0** :tada:
+
+The new Saros for Eclipse 16.0.0 contains multiple [bug fixes](#bug-fixes)
+and [improvements](#improvements).
+
+## Installation
+Saros for Eclipse 16.0.0 requires
+ - `JDK 8` or newer
```

Comment:
Specify supported Eclipse versions?

Reply:
This is a good question. The oldest version listed in the [eclipse marketplace](https://marketplace.eclipse.org/content/saros-distributed-collaborative-editing-and-pair-programming#group-details) is " Eclipse Neon", but even if we switched to Neon as the development environment, we should be compatible with 4.* (probably also 3.* but this is nothing I would like to test).
We have to test this and also adjust the marketplace entry to >= 4.

Reply:
Now I remember. We are compatible with >= 4.6 (Neon). Neon was the [first eclipse with JDK 8 runtime support](https://wiki.eclipse.org/Eclipse/Installation).